pkg/pointerdb/config.go

```diff
@@ -25,7 +25,7 @@ const (
 // PointerDB responsibility
 type Config struct {
 	DatabaseURL          string `help:"the database connection string to use" default:"bolt://$CONFDIR/pointerdb.db"`
-	MinInlineSegmentSize int64  `default:"1240" help:"minimum inline segment size"`
+	MinRemoteSegmentSize int    `default:"1240" help:"minimum remote segment size"`
```

Comment:
Probably doesn't matter right now, but in the future we probably need something for backwards compatibility with old config variables.